doc/nrf/libraries/bluetooth_services/rpc.rst

```diff
@@ -76,3 +80,19 @@ Then, you can invoke build command like this:
    :class: highlight
 
    west build -b *board* -- -DCONFIG_OVERLAY=my_overlay_file.conf
+
+API documentation
+*****************
+
+This library does not define a new API.
+Instead, it uses Zephyr's Bluetooth API.
+The behavior of the implementation is almost the same as Zephyr's with the following exceptions:
+
+* The latency is longer because of the overhead for exchanging messages between cores.
+* The :c:func:`bt_gatt_cancel` function is not implemented.
+* The ``flags`` field of  the :c:struct:`bt_gatt_subscribe_params` structure is atomic, so it cannot be correctly handled by the nRF RPC.
+  The library implements the following workaround for it:
+
+  * All ``flags`` are sent to the network core when either the :c:func:`bt_gatt_subscribe` or :c:func:`bt_gatt_resubscribe` function is called.
+     This covers most of the cases, because the ``flags`` are normally set once before those functions calls.
```

Comment:
```suggestion
  * All ``flags`` are sent to the network core when either the :c:func:`bt_gatt_subscribe` or :c:func:`bt_gatt_resubscribe` function is called.
    This covers most of the cases, because the ``flags`` are normally set once before those functions calls.
```

Reply:
This one extra space made the doc render incorrectly, sorry my bad.